lib/datadog/tracing/contrib/grpc/datadog_interceptor/client.rb

```diff
@@ -44,6 +44,9 @@ def annotate!(trace, span, metadata, call)
               host, _port = find_host_port(call)
               span.set_tag(Tracing::Metadata::Ext::TAG_PEER_HOSTNAME, host) if host
 
+              deadline = find_deadline(call)
+              span.set_tag(Ext::TAG_CLIENT_DEADLINE, (deadline.to_time.to_f * 1000).to_i) if deadline
```

Comment:
👋 @nvh0412, could you move the time conversion logic of deadline (from object to number) to the `find_deadline` method?

I think it looks cleaner if the caller here only has to set the deadline, like so:
```suggestion
              span.set_tag(Ext::TAG_CLIENT_DEADLINE, deadline) if deadline
```